README.md

```diff
@@ -45,34 +45,32 @@ target_os = []
 
 ### Build
 #### Windows
-- Set environmental variable `BOOST_ROOT` to your boost source tree.
-- Run `gclient sync`. It may take a long time to download large amount of data.
-- Go to src/srcitps/ directory, and run: `python build-win.py --gn_gen --sdk --tests --ssl_root /path/to/ssl --msdk_root /path/to/msdk --output_path /path/to/out`. The built binary will be under `output_path`, the document for sdk will also be copied to this directory if docs has been generated. If not set `output_path`, the built binary will be under src/out directory. Be noted the first time you run this would take a long time to pull chromium/webrtc dependencies and require a network accessible to Google's code/storage infrastructure. `ssl_root` to the directory of your OpenSSL 1.1.0 binary. `msdk_root` to the directory of your Intel Media SDK for Windows, version 2018 R1 or higher. `gn_gen` to generate args.gn, you need to add `--gn_gen` at the first time build or when you changed `msdk_root` and `ssl_root` path.
+- Set the environment variable `BOOST_ROOT` to your boost source tree.
```

Comment:
Not sure if `BOOST_ROOT` is still needed. Socket.IO depends on it. OWT doesn't depend on it directly. Anyway, this is not the part this patch is focus on.